package.json

```diff
@@ -34,7 +34,7 @@
     "lsif-protocol": "0.4.3",
     "minimist": "^1.2.0",
     "npm": "^6.14.2",
-    "typescript-lsif": "https://github.com/dbaeumer/TypeScript/releases/download/release%2Flisf%2F3.5.2/typescript-3.5.2-lsif.tgz",
+    "typescript-lsif": "https://github.com/dbaeumer/TypeScript/releases/download/3.8.3-lsif/typescript-3.8.3-lsif.tgz",
```

Comment:
Is this just pulling in the source? If yes, you could change this to a GitHub reference, and I think Renovate would be able to update it :)
```suggestion
    "typescript-lsif": "dbaeumer/TypeScript#3.8.3-lsif",
```

Reply:
Could you take a glance at dbaeumer's fork and confirm this would be alright to do? I'm a bit hesitant to touch this area as it's _really_ undocumented.

Our team sync has an agenda item to deal with this fork in a more permanent way (mirroring, maintaining our own fork, rewriting the indexer so it doesn't need the additional changes to the typescript codebase, etc).